GUI/Controls/GLViewerControl.cs

```diff
@@ -372,7 +374,8 @@ private static void OnDebugMessage(DebugSource source, DebugType type, int id, D
         private static readonly DebugProc OpenGLDebugMessageDelegate = OnDebugMessage;
 
         public Framebuffer GLDefaultFramebuffer;
-        public Framebuffer MainFramebuffer;
+        public Framebuffer BasePassFramebuffer;
+        public Framebuffer FinalFramebuffer;
```

Comment:
Maybe we can postprocess directly from `MainFramebuffer` into the `GLDefaultFramebuffer`.